cmake: finding HarfBuzz is broken

```
CMake Error at cmake/FindHarfBuzz.cmake:61 (include):
  include could not find load file:

    /deps/allegro5-dos/cmake/SelectLibraryConfigurations.cmake
Call Stack (most recent call first):
  addons/CMakeLists.txt:97 (find_package)


-- Could NOT find HarfBuzz (missing:  HARFBUZZ_LIBRARY)
```

https://github.com/liballeg/allegro5/blob/a40d30e21802ecf5c9382cf34af9b01bd3781e47/cmake/FindHarfBuzz.cmake#L61

There's no `cmake/SelectLibraryConfigurations.cmake` file in the repository.

--- a/cmake/FindHarfBuzz.cmake
+++ b/cmake/FindHarfBuzz.cmake
@@ -34,10 +34,6 @@
 set(HARFBUZZ_FIND_ARGS
   HINTS
     ENV HARFBUZZ_DIR
-  PATHS
-    ENV GTKMM_BASEPATH
-    [HKEY_CURRENT_USER\\SOFTWARE\\gtkmm\\2.4;Path]
-    [HKEY_LOCAL_MACHINE\\SOFTWARE\\gtkmm\\2.4;Path]
 )
 
 find_path(
@@ -58,8 +54,6 @@ if(NOT HARFBUZZ_LIBRARY)
     PATH_SUFFIXES
       lib
   )
-  include(${CMAKE_CURRENT_LIST_DIR}/SelectLibraryConfigurations.cmake)
-  select_library_configurations(HARFBUZZ)
 else()
   # on Windows, ensure paths are in canonical format (forward slahes):
   file(TO_CMAKE_PATH "${HARFBUZZ_LIBRARY}" HARFBUZZ_LIBRARY)

